src/game/state.ts

```diff
@@ -15,3 +17,17 @@ export const initGameState = (): GameState => ({
   turnNumber: 0,
   message: "Welcome to Kamon 🍱 ! Black player, you turn",
 });
+
+export const checkIfGameWon = (
```

Comment:
Is this function  intend to check other conditions ? Or it only check if no move are possible ? if that is the case, maybe rename it in `isThereMoveLeft` ?

Reply:
We probably need to change this to `isThereMoveLeft` and make a distinct `checkIfGameWon` function. It would check all 3 victory conditions by calling an helper function for each one of them.

Reply:
Done !